types/tailwindcss/test/tailwindcss-optional-config-tests.ts

```diff
@@ -0,0 +1,5 @@
+import type { TailwindConfig } from '../tailwind-config';
```

Comment:
The relative import is whats causing the CI to fail, here's the fix (haven't tested it):
```suggestion
import type { TailwindConfig } from 'tailwindcss';
```